core/src/Cabana_ParticleInit.hpp

```diff
@@ -132,6 +137,1183 @@ void createRandomParticles( PositionType& positions,
                            box_max );
 }
 
+/*!
+  Populate the particles based on the description of the distribution function
+  in gaussians
+*/
+template <typename CellSliceType, typename WeightSliceType,
+          typename PositionSliceType, typename VelocitySliceType,
+          typename GaussianType>
+size_t initializeRandomParticles( CellSliceType& cell, WeightSliceType& macro,
```

Comment:
I believe the conflicts with master are related to the rewrite of particle init. The changes were primarily to match the structured-grid side of the library, where each free function is now `createParticles( CreationTypeTag, ... )`: currently only `InitRandom` available, but this seems like it should become `createParticles( InitGaussian, ... )`